dev/com.ibm.ws.webcontainer.security/src/com/ibm/websphere/security/web/PasswordExpiredException.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.websphere.security.web;
+
+import javax.servlet.ServletException;
+/**
+ *
+ */
+public class PasswordExpiredException extends ServletException{
+
+    private static final long serialVersionUID = 1L;
+
+    public PasswordExpiredException(String message) {
```

Comment:
We need to be captain obvious here and say something in the javadoc like "Create a new PasswordExpiredException with a message".  same for the other constructors, and for the user revoked exception too.